src/esp/gfx/Renderer.cpp

```diff
@@ -79,7 +82,27 @@ struct Renderer::Impl {
                 esp::gfx::Renderer::Impl::RendererShaderType::
                     DepthTextureVisualizer);
 
+#ifdef CORRADE_TARGET_APPLE
+        Mn::Image2D image = tgt.getDepthTexture().image(
+            0, {Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float});
+        Mn::ImageView2D imgView{image.storage(), Mn::PixelFormat::R32F,
+                                image.size(), image.data()};
+
+        if ((visualizedTex_ == Cr::Containers::NullOpt) ||
+            (visualizedTex_ != Cr::Containers::NullOpt &&
+             visualizedTex_->imageSize(0) != image.size())) {
```

Comment:
Can be just 

```suggestion
        if (visualizedTex_ == Cr::Containers::NullOpt ||
             visualizedTex_->imageSize(0) != image.size()) {
```

due to the short-circuit evaluation of the boolean operators. Or even

```suggestion
        if (!visualizedTex_ ||
             visualizedTex_->imageSize(0) != image.size()) {
```